### DISCUSSION TITLE
How to Translate the Admin Dashboard to Spanish?

### DISCUSSION THREAD

**Discussion by Pacocc97** [medusajs/medusa#9996] [Q&A] [answered]
Hello everyone,

I can’t find a way to translate the admin dashboard to Spanish. Does Medusa have any native options for this? Or how can I manually translate the UI?

I really need this feature.

Thanks!

**Comment by SalahAdDin** [comment]
> Hello everyone,
> 
> I can’t find a way to translate the admin dashboard to Spanish. Does Medusa have any native options for this? Or how can I manually translate the UI?
> 
> I really need this feature.
> 
> Thanks!

If you have notions of translation with React, you can help by improving the translation files, but as far as I know, as the current version is in constant changes, all literals could change in any time.

**Comment by shahednasser** [comment, marked as answer]
@Pacocc97 we've now added to the documentation a guide on contributing translation for the admin:

https://docs.medusajs.com/resources/contribution-guidelines/admin-translations

When you contribute the translation, it will be included in our next release and you can use it then in the dashboard.
